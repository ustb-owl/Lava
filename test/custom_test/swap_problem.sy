int main() {
    int a;
    int b;
    int c;
    int i;
    a = 1;
    b = 2;
    i = 0;
    while (i < 9) {
        putint(a);
        c = a;
        a = b;
        b = c;
        i = i + 1;
    }
    return a;
}