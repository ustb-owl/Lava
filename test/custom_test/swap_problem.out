121212121
2